nano/secure/utility.cpp

```diff
@@ -60,15 +63,6 @@ void nano::remove_temporary_directories ()
 		{
 			std::cerr << "Could not remove temporary directory: " << ec.message () << std::endl;
 		}
-
```

Comment:
This is a serious change of behaviour and not a logging change.
I personally agree with the change but it should not had been part of the logging PR because this affects node operators.

Reply:
Can you explain how this affects node operators? I only saw this function used in unit testing. Generally there was badly implemented code, fixing badly implemented code that culminated in some strange bugs when I needed to ensure orderly node initialization.